21.2.1 broke NetBSD source build

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

21.2.0 is fine but, 21.2.1 breaks our builds, can't update package.

### Theme

NA

### What OS are you seeing the problem on?

_No response_

### Which shell are you using?

elvish

### Log output

```Shell
# github.com/jandedobbeleer/oh-my-posh/src/cli
cli/upgrade.go:27:12: undefined: upgrade.Run
cli/upgrade.go:42:11: undefined: upgrade.Run
*** Error code 1

Stop.
make[1]: stopped in /usr/pkgsrc/wip/oh-my-posh
*** Error code 1

Stop.
```

--- a/src/upgrade/cli_other.go
+++ b/src/upgrade/cli_other.go
@@ -1,4 +1,4 @@
-//go:build linux || freebsd || openbsd
+//go:build linux || freebsd || openbsd || netbsd
 
 package upgrade
 

